srcpkgs/ncspot/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.1.4
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config python3"
+makedepends="pulseaudio-devel dbus-devel libressl-devel ncurses-devel"
+depends="pulseaudio dbus libxcb libssl48 ncurses"
+short_desc="Cross-platform ncurses Spotify client written in Rust"
+maintainer="Rien Maertens <rien.maertens@posteo.be>"
+archs="i686 x86_64 x86_64-musl"
```

Comment:
Why is it restricted to these? If cross compiling isn't working, you probably want `nocross=yes`, instead.

Reply:
[The other archs fail](https://travis-ci.org/github/void-linux/void-packages/builds/705378524), I could also use the same construct as spotifyd if that's desired?
```
case "$XBPS_TARGET_MACHINE" in
	aarch64-musl) broken="https://travis-ci.org/void-linux/void-packages/jobs/..." ;;
esac
```

Reply:
If it's broken specifically for one arch that's fine, but if it works when compiling natively, but not when cross compiling, it should be marked as nocross. Usually you use archs only when it's a platform or ISA level support thing. If it's broken on all cross compiling cases, the issue is likely with the "cross" part

Reply:
Error is `Os { code: 8, kind: Other, message: "Exec format error" }`, so this *is* cross build issue, and have good chance to be build natively on arm. Please mark nocross.

Reply:
Done.